lib/logstash/inputs/jdbc.rb

```diff
@@ -137,6 +137,9 @@ class LogStash::Inputs::Jdbc < LogStash::Inputs::Base
   # Whether to save state or not in last_run_metadata_path
   config :record_last_run, :validate => :boolean, :default => true
 
+  # Whether to force the lowercasing of identifier fields
+  config :force_lowercase_keys, :validate => :boolean, :default => true
```

Comment:
according to https://github.com/logstash-plugins/logstash-input-jdbc/issues/44#issuecomment-164436249 shouldn't this be false?